### DISCUSSION TITLE
How to correctly use polygon_threshold?

### DISCUSSION THREAD

**Discussion by xithalius** [osm-search/Nominatim#3000] [Q&A] [answered]
I'm trying to use the polygon_threshold param to hopefully increase performance and reduce payload size but it's not working.
I'm getting a 500 "Could not get outline".

Test URL
`https://nominatim.openstreetmap.org/reverse?lat=51.0676813&lon=4.4572147&format=json&addressdetails=0&polygon_geojson=1&polygon_threshold=0.005&accept-language=en&zoom=10`

**Comment by mtmail** [comment, marked as answer]
You're using it correct. I see the same error with Nominatim 4.2 (`Nominatim version 4.1.99-0 (9c152a03)`). Can you file it as an issue? https://github.com/osm-search/Nominatim/issues

The /lookup endpoint works for the same place so it's unlikely that the OSM data is broken. That might be a work-around for you.

https://nominatim.openstreetmap.org/lookup?osm_ids=R3369539&format=json&addressdetails=0&polygon_geojson=1&polygon_threshold=0.005&accept-language=en

**Reply by xithalius** [reply]
Thank you for the fast response :-)
